meeting-notes/2019/Q1/2019-02-11--go-core-dev-team-weekly.md

```diff
@@ -0,0 +1,169 @@
+# 💫Golang Core Dev Team Weekly Sync 🙌🏽 2019-02-11
+
+Please note. There has been a  [meeting format change](https://github.com/ipfs/team-mgmt/issues/827#issuecomment-452157617) intended to make our brief sync time more valuable and more easily understood by new/occasional attendees.
+
+**Meeting Lead:** @eingenito
+
+**Attendees:**
+  - @hannahhoward
+  - @jimpick
+  - @aschmahmann
+  - @eingenito
+  - @michaelavila
+  - @stebalien
+  - @bigs
+  - @djdv
+  - @kubuxu
+  - @mgoelzer
+  - @magik6k
+  - @warpfork
+  
+**Recording:** [Recordings are available to the core team only](https://drive.google.com/drive/u/2/folders/1iyFnbY4RRsfIcS_b6M9wMEjbuZceSw9z)
+
+
+## Agenda
+- Review Announcements for the whole team
+- Review blockers and needs from other team members (not general updates)
+  - Are you blocked waiting for input?
+  - PRs for review? Remember the [project board](https://github.com/orgs/ipfs/projects/1)
+- Review progress on major initiatives
+  - Find or create the section for the IPFS initiative(s) you worked on.
+  - Summarize the initiative briefly in notes to give a little context.
+  - Summarize your progress
+  - Summarize your next steps.
+- Review other Questions/Announcements/Notes
+
+## Current IPFS Initiatives
+#### Datastore
+Summary: We are investigating various IPFS datastore options with the goal of replacing the current default datastore with a faster, more scalable one: [Epic Issue](https://github.com/ipfs/go-ipfs/issues/4279)
+
+@eingenito - This issue is blocked (ish?) on benchmarking infrastructure. There may be several other outstanding issues that need to be addressed before we even have a go/no-go on Badger. Epic may need a refresh to clean out closed issues and add new ones. 
+
+@magik6k - Moved go-ds-bench a bit forward, can start running serious benchmarks soon
+  - Needs infra: https://github.com/protocol/ad-hoc-infra/pull/24
+    - This is dev-only, will need more & more diverse infra soon
```

Comment:
@eingenito @magik6k Is there a requirements doc for what this benchmarking infrastructure is? Would providing 4TB to each member of the team enable you to start pulling some results?

Reply:
This is a small scale infrastructure where I want to test a diverse set of configurations, to tinker with tooling and different performance characteristics, mostly to know where 'more testing is needed'.

Testing on large data can produce interesting outcomes, but tends to be predictable from small scale data. I want to start by testing a lot of scenarios to see what will tell us the most when scaled up